### ISSUE TITLE
ready function on package

### ISSUE THREAD

**Issue by amelon** [chrisjpowers/air-drop#5]
Hi,

as `include` and `require` use a deferred execution to build path, it would be usefull (at least for test) to have a `ready` function to ensure package is ready to answer correctly.

Using mocha I've done following "trick" 

```
function waitPathReady(package, done) {
  if (package.paths.length) return done();
  console.log('nb_waits', ++nb_waits);
  setTimeout(function() {waitPathReady(package, done)}, 5);
}
```

Or maybe, package should answer once it is ready?

**Comment by chrisjpowers** [comment]
I ran into the same problem when I was building the command line tool -- I wanted to use the package once it's ready, but didn't have a good way of doing it. I like the idea of a `ready` callback, I'll work on that.
